src/object_schema.cpp

```diff
@@ -107,11 +108,31 @@ Property *ObjectSchema::property_for_name(StringData name) {
     return nullptr;
 }
 
-const Property *ObjectSchema::property_for_name(StringData name) const {
+Property *ObjectSchema::property_for_alias(StringData alias)
+{
+    for (auto& prop : persisted_properties) {
+        if (StringData(prop.alias.empty() ? prop.name : prop.alias) == alias)
+            return &prop;
+    }
+    for (auto& prop : computed_properties) {
+        if (StringData(prop.alias.empty() ? prop.name : prop.alias) == alias)
```

Comment:
Supporting both an alias and a name for computed properties doesn't make much sense, since there is no associated column name for computed properties.

Reply:
True, the alias and name seem to be the same thing for computed properties. However, supporting it doesn't seem to be directly harmful, whereas not supporting it would complicate using the schema API.